spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnRef.scala

```diff
@@ -17,6 +17,9 @@ sealed trait NamedColumnRef extends ColumnRef {
   /** Returns a name of the selection as it is seen in the result set. Most likely this is going to be
     * used when providing custom column name to field name mapping. */
   def selectedAs: String
```

Comment:
`selectedAs` and `alias` are confusing. At first I thought that `selectedAs` applies to CassandraRDD select. However, in fact, this is how we identify this column in the ResultSet coming from Cassandra. So maybe rename `selectedAs` to something more unambiguous, like `resultSetColumnName` or `selectedFromCassandraAs`?